pyseir/cli.py

```diff
@@ -50,6 +50,20 @@ def _generate_whitelist():
     gen.generate_whitelist()
 
 
+def _generate_infection_rate_metric(regions: List[pipeline.Region]) -> pd.DataFrame:
```

Comment:
I think the type is meant to be `List[infer_rt.RegionalInput]`

Reply:
Yep. Fixed.